ssl/ssl_test.cc

```diff
@@ -5758,18 +5758,19 @@ TEST(SSLTest, EmptyCipherList) {
   // Configuring the empty cipher list with |SSL_CTX_set_cipher_list|
   // succeeds.
   EXPECT_TRUE(SSL_CTX_set_cipher_list(ctx.get(), ""));
-
   // The cipher list is updated to empty.
   EXPECT_EQ(0u, sk_SSL_CIPHER_num(SSL_CTX_get_ciphers(ctx.get())));
 
+  // Configuring the empty cipher list with |SSL_CTX_set_ciphersuites|
+  // also succeeds.
+  EXPECT_TRUE(SSL_CTX_set_ciphersuites(ctx.get(), ""));
+  EXPECT_EQ(0u, sk_SSL_CIPHER_num(SSL_CTX_get_ciphers(ctx.get())));
```

Comment:
Thanks, LGTM with this change, and should detect any regressions in how MySQL+MariaDB use this :raised_hands:

Reply:
Thanks, this is good.

This _doesn't actually verify_ the pattern used by MySQL+MariaDB :sweat_smile: , which is:

1. Do *both* `SSL_CTX_set_cipher_list("non empty list for TLSv1.2")` *and* `SSL_CTX_set_ciphersuites("empty list for TLSv1.3")`
2. Expect that there will be _only_ TLSv1.2 (or earlier) ciphers available for new connections, and no TLSv1.3 ciphersuites

As described in https://github.com/openssl/openssl/issues/13704#issuecomment-748921021, it seems like it's quite complicated to get a list of _only_ the TLSv1.3 ciphersuites or _only_ the TLSv1.2 ciphers :frowning_face: